umm-spec-lib/src/cmr/umm_spec/spatial_conversion.clj

```diff
@@ -53,20 +61,26 @@
    "Minimum Altitude"
    "Minimum Depth"])
 
+(def upcase-valid-vertical-spatial-domain-types
+  "Upcase values to allow for case-insenitive validation"
+  (map util/safe-uppercase valid-vertical-spatial-domain-types))
+
 (defn vertical-spatial-domain-type-is-valid?
   "Return true or false based on whether or not the given value matches
    one of the valid values in valid-vertical-spatial-domain-types."
   [vs-domain-type]
-  (some #(= vs-domain-type %) valid-vertical-spatial-domain-types))
+  (some #(= (util/safe-uppercase vs-domain-type) %)
+        upcase-valid-vertical-spatial-domain-types))
 
 (defn drop-invalid-vertical-spatial-domains
   "Any VerticalSpatialDomain with a Type not in valid-vertical-spatial-domain-types
    is not considered valid according to UMM spec v1.10.0 and should be dropped.
    This behavior will eventually generate errors"
   [vertical-spatial-domains]
-  (seq (filter
-         #(vertical-spatial-domain-type-is-valid? (:Type %))
-         vertical-spatial-domains)))
+  (->> vertical-spatial-domains
+       (filter #(vertical-spatial-domain-type-is-valid? (:Type %)))
+       (map #(update % :Type util/capitalize-words))
+       seq))
```

Comment:
Please combining the functions the same way as the tiling id case I mentioned above.  Use the more generic match-enum-case, rather than capitalize-words.